tests/config/Shared/config_local.php

```diff
@@ -0,0 +1,63 @@
+<?php
```

Comment:
do we really need local config?
probably naming is wrong

Reply:
Yes, it's required to setup fake project